0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -137,11 +187,12 @@ HTTP/1.1 201 Created
 Content-Type: application/json
 
 {
-  "receipt_condition": "cc:1:1:47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU:1"
+  "packet": "asdglkjlsdfoiuaosiduf...",
+  "condition": "cc:0:3:47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU:0"
 }
```

Comment:
@justmoon If the `condition` is communicated separately from the `packet` should this be some binary format to prevent having to encode and decode base64?